### PR TITLE
[4.0] Use namespaced classes for layouts [Part 8]

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Use namespaced classes for `layouts/joomla/form/**`

### Testing Instructions

code review

### CODE REVIEW

File: layouts/joomla/form/field/subform/repeatable-table.php [hunk 1]
```diff
@@ -83,7 +87,7 @@
 					<th style="width:8%;">
 					<?php if (!empty($buttons['add'])) : ?>
 						<div class="btn-group">
-							<a class="group-add btn btn-sm button btn-success" aria-label="<?php echo JText::_('JGLOBAL_FIELD_ADD'); ?>"><span class="icon-plus icon-white" aria-hidden="true"></span> </a>
+							<a class="group-add btn btn-sm button btn-success" aria-label="<?php echo Text::_('JGLOBAL_FIELD_ADD'); ?>"><span class="fa fa-plus icon-white" aria-hidden="true"></span> </a>
```
Comment (Quy) [inline layouts/joomla/form/field/subform/repeatable-table.php:90]: Just checking if `icon-white` is still valid.

Comment (C-Lodder) [reply layouts/joomla/form/field/subform/repeatable-table.php:90]: I believe I added that myself when I started on Atum, it's valid

Comment (Quy) [reply layouts/joomla/form/field/subform/repeatable-table.php:90]: I removed `icon-white` on a button in the backend and it appears to be fine. Maybe it inherited the link color.

Comment (C-Lodder) [reply layouts/joomla/form/field/subform/repeatable-table.php:90]: I'm leaving as is for now. I only changed a few icons but technically out of scope of this PR anyway

**Pull request merged**